packages/ibm-products-web-components/src/components/page-header/page-header.stories.ts

```diff
@@ -0,0 +1,480 @@
+
+const tags = [
+  {
+    type: 'blue',
+    text: 'Tag 1',
+    size: 'md',
+  },
+  {
+    type: 'purple',
+    text: 'Tag 2',
+    size: 'md',
+  },
+  {
+    type: 'red',
+    text: 'Tag 3',
+    size: 'md',
+  },
+  {
+    type: 'blue',
+    text: 'Tag 4',
+    size: 'md',
+  },
+  {
+    type: 'purple',
+    text: 'Tag 5',
+    size: 'md',
+  },
+  {
+    type: 'red',
+    text: 'Tag 6',
+    size: 'md',
+  },
+];
+
+const args = {
+  border: true,
+  pageActionsFlush: false,
+  contentActionsFlush: false,
+  title:
+    'Virtual-Machine-DAL-really-long-title-example-that-goes-at-least-2-lines-long',
+  renderBreadcrumbIcon: true,
+};
+
+const argTypes = {
+  border: {
+    description:
+      'Specify whether to render `c4p-page-header-breadcrumb` border',
+    control: 'boolean',
+  },
+  pageActionsFlush: {
+    description:
+      'Specify whether the page actions within `c4p-page-header-breadcrumb` should be flush',
+    control: 'boolean',
+  },
+  contentActionsFlush: {
+    description:
+      'Specify whether the content actions within `c4p-page-header-breadcrumb` should be flush with the page actions',
+    control: 'boolean',
+  },
+  title: {
+    description:
+      'Provide the title text to be rendered within  `c4p-page-header-content`',
+    control: 'text',
+  },
+  renderBreadcrumbIcon: {
+    description:
+      'Specify whether to render the `c4p-page-header-breadcrumb` icon (storybook control only)',
+    control: 'boolean',
+  },
+};
+
+export const Default = {
+  args,
+  argTypes,
+  render: (args) => {
+    const {
+      border,
+      pageActionsFlush,
+      contentActionsFlush,
+      title,
+      renderBreadcrumbIcon,
+    } = args ?? {};
+    return html`
+      <c4p-page-header>
+        <c4p-page-header-breadcrumb
+          border="${border}"
+          ?page-actions-flush="${pageActionsFlush}"
+          ?content-actions-flush="${contentActionsFlush}"
+        >
+          ${renderBreadcrumbIcon ? Bee16({ slot: 'icon' }) : undefined}
+          <cds-breadcrumb>
+            <cds-breadcrumb-item>
+              <cds-breadcrumb-link href="#">Breadcrumb 1</cds-breadcrumb-link>
+            </cds-breadcrumb-item>
+            <cds-breadcrumb-item>
+              <cds-breadcrumb-link href="#">Breadcrumb 2</cds-breadcrumb-link>
+            </cds-breadcrumb-item>
+          </cds-breadcrumb>
+          <div slot="content-actions">
+            <cds-button size="md">Button</cds-button>
+          </div>
+          <cds-icon-button slot="page-actions" kind="ghost" size="md"
+            >${Activity16({ slot: 'icon' })}</cds-icon-button
+          >
+          <cds-icon-button slot="page-actions" kind="ghost" size="md"
+            >${AiGenerate16({ slot: 'icon' })}</cds-icon-button
+          >
+          <cds-icon-button slot="page-actions" kind="ghost" size="md"
+            >${CloudFoundry16({ slot: 'icon' })}</cds-icon-button
+          >
```

Comment:
Only caught this because I was on the Docs page, but these should probably have `align="bottom"`.

![image](https://github.com/user-attachments/assets/ed24681c-c580-4e2a-8694-5980d4e986d0)